dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentCDIApp/src/concurrent/mp/fat/cdi/web/MPConcurrentCDITestServlet.java

```diff
@@ -671,7 +671,8 @@ public void testTransactionContextPropagation() throws Exception {
             assertEquals(Status.STATUS_ACTIVE, tx.getStatus());
 
             Future<?> f = executor.submit(() -> System.out.println("Should not be able to submit this task."));
-            fail("Submitted task from within a transaction when transaction context propagation is enabled: " + f);
+            // TODO fail("Submitted task from within a transaction when transaction context propagation is enabled: " + f);
+            f.get(TIMEOUT_MIN, TimeUnit.MINUTES);
         } catch (UnsupportedOperationException x) {
             if (x.getMessage() == null || !x.getMessage().startsWith("CWWKC1157E"))
```

Comment:
Yes, because I expect this will be switched back.  Transaction context propagation is experimental, and how much of it will be supported is unclear at this point.